lib/rules/jsx-indent-props.js

```diff
@@ -124,10 +124,14 @@ module.exports = {
 
       const indent = regExp.exec(src);
       const useOperator = /^([ ]|[\t])*[:]/.test(src) || /^([ ]|[\t])*[?]/.test(src);
+      const useBracket = /^([ ]|[\t])*[<]/.test(src);
```

Comment:
it occurs to me that perhaps we could use eslint AST utilities rather than regexes to detect these things. have you explored that at all?

Reply:
First, thanks for the quick feedback!
I haven't explored so far, can I ask where I can get more information about it?

Reply:
There's https://github.com/jsx-eslint/jsx-ast-utils, and https://eslint.org/docs/developer-guide/working-with-rules that might help.

I'll land this as-is for now, and we can improve it in a followup.

Reply:
@ljharb 
Checking if there are other test cases with indentation error,
I will follow up with reference to what you have informed me.

Reply:
Sounds good!